components/backends/vllm/src/dynamo/vllm/publisher.py

```
@@ -1,6 +1,7 @@
 # SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+from optparse import Option
```

Ruff (0.12.2)

`optparse.Option` imported but unused

Remove unused import: `optparse.Option`

(F401)